README.md

```diff
@@ -11,153 +11,43 @@ its proof of work (POW). Taking development cues from Tenebrix and Litecoin,
 Dogecoin currently employs a simplified variant of scrypt.
 - **Website:** [dogecoin.com.](https://dogecoin.com)
 
-## License – Much license ⚖️
-Dogecoin Core is released under the terms of the MIT license. See
-[COPYING](COPYING) for more information or see
-[opensource.org](https://opensource.org/licenses/MIT)
```

Comment:
License definitely has to stay, we need to ensure it's highly visible in a number of places so we're covered.

Reply:
The license section is at the bottom of the page instead of the top.

I already [answered](https://github.com/dogecoin/dogecoin/pull/2313#issuecomment-864531077) to you on previous [comments](https://github.com/dogecoin/dogecoin/pull/2313#issuecomment-864528521), didn't get feedback :)